README.md

```diff
@@ -94,8 +94,8 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 ## Frontend
 
   - :video_camera: [Full Course](https://www.youtube.com/watch?v=zJSY8tbf_ys)
-
-  ### HTML
+  
+    ### HTML
```

Comment:
I can some extra space here, Please remove that space and make sure it is perfectly aligned with others in the section.

Reply:
Now it is properly aligned
![s1](https://user-images.githubusercontent.com/84179065/189785531-0964fa8c-8e2e-40a1-aed4-cc0918aa6e1f.JPG)